app/components/consultee_status_component.html.erb

```diff
@@ -0,0 +1 @@
+<%= consultee_status %>
```

Comment:
Is there a reason you're not using the `base_component.html.erb` here?

Reply:
Because I didn't realise it existed, lol. Will update.

Reply:
... actually, I've realised I can't now I've looked at it, as I have more statuses to account for than are in the `base_component`. So that either means :
- loading this with more statuses
- having more conditionals in the table column
- using the status tag

Any of these would work - not sure which would be best?

Reply:
I think there are maybe 5 new ones you'd be introducing? I'd chuck them in the table, that's what I've been doing. I think (I hope) all these components will get a nice refactor soon, so easier to sellotape for now 😆

Reply:
OK cool 😎